packages/core/src/Modifier.ts

```diff
@@ -8,6 +8,7 @@ export interface Modifier {
     constructor: ModifierConstructor & this;
 }
 export class Modifier {
+    preserve: boolean = true;
```

Comment:
Is the typing necessary? Doesn't typescript recognize that setting the property to `true` means its type is boolean? Or does it think the type is `true` then?

Reply:
No it's not necessary. I think our prettier config fixes that kind of things automatically.